include/openthread/netdiag.h

```diff
@@ -201,6 +208,29 @@ typedef struct otNetworkDiagMacCounters
     uint32_t mIfOutDiscards;
 } otNetworkDiagMacCounters;
 
+/**
+ * Represents a Network Diagnostics MLE Counters value.
+ *
+ */
+typedef struct otNetworkDiagMleCounters
+{
+    uint16_t mDisabledRole;                  ///< Number of times device entered disabled role.
+    uint16_t mDetachedRole;                  ///< Number of times device entered detached role.
+    uint16_t mChildRole;                     ///< Number of times device entered child role.
+    uint16_t mRouterRole;                    ///< Number of times device entered router role.
+    uint16_t mLeaderRole;                    ///< Number of times device entered leader role.
+    uint16_t mAttachAttempts;                ///< Number of attach attempts while device was detached.
+    uint16_t mPartitionIdChanges;            ///< Number of changes to partition ID.
+    uint16_t mBetterPartitionAttachAttempts; ///< Number of attempts to attach to a better partition.
+    uint16_t mParentChanges;                 ///< Number of time device changed its parent.
+    uint64_t mTrackedTime;                   ///< Milliseconds tracked by next counters (zero if not supported)
```

Comment:
```suggestion
    uint64_t mTrackedTime;                   ///< Milliseconds tracked by next counters (zero if not supported).
```

Reply:
Is this expected to equal the sum of the following counters?

Reply:
Yes. It is sort of redundant but mirrors the definition of `otMleCounters`.